java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1512,22 +1511,6 @@ public static Table merge(List<Table> tables, OrderByArg... args) {
     return merge(tables.toArray(new Table[tables.size()]), args);
   }
 
-  public static OrderByArg asc(final int index) {
```

Comment:
Seems like we could avoid breaking downstream project compilation by using a deprecated, trivial derivation here from the new `OrderByArg` class.

Reply:
It would still not be source compatible.  The return type would have changed, and cannot make the moved OrderByArg class backwards compatibility because I had to completely move it out of Table to avoid loading the native libraries.

Reply:
We also had OrderByArg marked as final to avoid others from sub-classing it. If we wanted to keep that, then we could not have something in Table extend the real class.

Reply:
Ah right, it doesn't solve the problem of methods that used to return the old type can't keep returning that old type.